service/worker/deletenamespace/fx.go

```diff
@@ -54,20 +65,36 @@ type (
 	}
 	componentParams struct {
 		fx.In
-		VisibilityManager manager.VisibilityManager
-		MetadataManager   persistence.MetadataManager
-		HistoryClient     resource.HistoryClient
-		MetricsHandler    metrics.Handler
-		Logger            log.Logger
+		ActivityWorkerConfig activityWorkerConfig
+		VisibilityManager    manager.VisibilityManager
+		MetadataManager      persistence.MetadataManager
+		HistoryClient        resource.HistoryClient
+		MetricsHandler       metrics.Handler
+		Logger               log.Logger
 	}
 )
 
-var Module = workercommon.AnnotateWorkerComponentProvider(newComponent)
+var Module = fx.Options(
+	fx.Provide(newActivityWorkerConfig),
+	workercommon.AnnotateWorkerComponentProvider(newComponent),
+)
+
+func newActivityWorkerConfig(
+	dc *dynamicconfig.Collection,
+) activityWorkerConfig {
+	return activityWorkerConfig{
+		maxConcurrentActivityExecutionSize: dc.GetIntProperty(dynamicconfig.WorkerDeleteNamespaceMaxConcurrentActivityExecutionSize, 0),
```

Comment:
let's use 4 as the default value, I think that's the default concurrency for deletion executions activity.
@yux0 are we using the default value or a concurrency value is explicitly specified when starting the delete namespace workflow?

cc @alexshtin

Reply:
I guess limiting either concurrent activity count or activity rps will achieve our goal of limiting total delete workflow rps across workflows. (assuming worker pod count and delete workflow rps per activity are constants)

With activity rps, we will also need the data for latency of each activity.